database/index.js

```diff
@@ -1,104 +1,110 @@
-const mongoose = require('mongoose');
-var { dbUrl, user, pass } = require('./config.js')
+var configObject = require('./config.js')
+const { Client } = require('pg');
+const client = new Client(configObject);
 
-mongoose.connect(dbUrl, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-  user: user,
-  pass: pass
-});
+client.connect();
 
-let tripSchema = mongoose.Schema({
-  locationId: {type: Number, required: true},
-  checkIn: {type: Date, required: true},
-  checkOut: {type: Date, required: true},
-  adults: Number,
-  children: Number,
-  rooms: {type: Number, required: true}
-});
 
-let locationSchema = mongoose.Schema({
-  locationId: {type: Number, required: true},
-  rooms: {type: Number, required: true},
-  name: String,
-  lowDays: [{}]
-});
-
-
-
-let Trip = mongoose.model('Trip', tripSchema);
-let Location = mongoose.model('Location', locationSchema);
+var createTrip = ({ locationid, checkin, checkout, adults, children, rooms }) => {
+  const text = 'INSERT INTO trips(locationid, checkin, checkout, adults, children, rooms) VALUES($1, $2, $3, $4, $5, $6)'
+  const values = [locationid, checkin, checkout, adults, children, rooms];
+  return new Promise((resolve, reject) => {
+    client
+      .query(text, values)
+      .then((res) => {
+        resolve(res);
+      })
+      .catch((err) => {
+        console.error('Error creating a new trip in database/index.js', err);
+        reject(err);
+      });
+  })
+};
 
-var createTrip = (trip) => {
+var getLowDays = (locationId) => {
+  const text = 'SELECT * FROM lowdays WHERE locationid=$1';
+  const value = [locationId];
   return new Promise((resolve, reject) => {
-    Trip.create(trip, (err, result) => {
-      if (err) {
+    client
+      .query(text, value)
+      .then((res) => {
+        resolve(res);
+      })
+      .catch((err) => {
         reject(err);
```

Comment:
Maybe add a console.error for more description as where the error occurred, like your other functions